### PR TITLE
fix box ellipsis

### PR DESCRIPTION
**Pull request by ThibodeauJF**
https://coveord.atlassian.net/browse/KIT-838

### CODE REVIEW

**Comment by ThibodeauJF** [comment]
I took a look at the ellipsis issue & opened a quick draft https://github.com/coveo/ui-kit/pull/1026, I'm also reusing the BoxValue since it's similar enough like Sami said.

For the checkbox, the box should be aligned when selected / deselected and not a little to the left. There also needs to have an hover effect (text primary & underline)

Screen Shot 2021-07-29 at 2 34 32 PM

**Comment by ThibodeauJF** [comment]
@nbuissoncoveo Feel free to integrate these changes into your PR!

File: packages/atomic/src/components/facets-v1/facet-value-label-highlight/facet-value-label-highlight.tsx [hunk 1]
```diff
@@ -9,7 +9,9 @@ export const FacetValueLabelHighlight: FunctionalComponent<FacetValueLabelHighli
     <span
       title={props.displayValue}
       part="value-label"
-      class={`value-label ellipsed ${props.isSelected ? 'font-bold' : ''}`}
+      class={`value-label ellipsed w-full ${
```
Comment (nbuissoncoveo) [inline packages/atomic/src/components/facets-v1/facet-value-label-highlight/facet-value-label-highlight.tsx:12]: I also was able to fix the ellipsis by assigning w-full :)

**Review by nbuissoncoveo — APPROVED** [review]

**Comment by ThibodeauJF** [comment]
I won't merge this, it was only a POC! 😄

**Comment by nbuissoncoveo** [comment]
> I won't merge this, it was only a POC! 😄

Oh ok I wanted to merge it into mine but no worries I will implement the facetValueBox changes.

**Pull request closed without merging**